src/parser/paladin/holy/modules/azeritetraits/LightsDecree.js

```diff
@@ -89,33 +98,21 @@ class LightsDecree extends Analyzer {
   }
```

Comment:
I believe totalHealing() is double counting beacon healing, because beacon healing will also trigger onHeal, it just won't pass through information about the healing source spell.  You can check this by adding a console.log that prints when on onHeal event is triggered from beacon of light's spell id.